I'm trying to build it locally but it's failing during a CMake FetchContent operation with :

```
CMake Error at /nix/store/q1nssraba326p2kp6627hldd2bhg254c-cmake-3.29.2/share/cmake-3.29/Modules/FetchContent.cmake:1714 (message):
  Build step for libcoap failed: 1
Call Stack (most recent call first):
  /nix/store/q1nssraba326p2kp6627hldd2bhg254c-cmake-3.29.2/share/cmake-3.29/Modules/FetchContent.cmake:1854:EVAL:2 (__FetchContent_directPopulate)
  /nix/store/q1nssraba326p2kp6627hldd2bhg254c-cmake-3.29.2/share/cmake-3.29/Modules/FetchContent.cmake:1854 (cmake_language)
  /nix/store/q1nssraba326p2kp6627hldd2bhg254c-cmake-3.29.2/share/cmake-3.29/Modules/FetchContent.cmake:2081 (FetchContent_Populate)
  3rdparty/libossia/cmake/deps/coap.cmake:37 (FetchContent_MakeAvailable)
  3rdparty/libossia/cmake/OssiaDeps.cmake:107 (include)
  3rdparty/libossia/CMakeLists.txt:36 (include)
```

--- a/maintainers/maintainer-list.nix
+++ b/maintainers/maintainer-list.nix
@@ -9393,6 +9393,12 @@
     github = "jceb";
     githubId = 101593;
   };
+  jcelerier = {
+    name = "Jean-Michaël Celerier";
+    email = "jeanmichael.celerier@gmail.com";
+    github = "jcelerier";
+    githubId = 2772730;
+  };
   jchw = {
     email = "johnwchadwick@gmail.com";
     github = "jchv";

